networking/private-cloud-function-from-onprem/main.tf

```diff
@@ -49,69 +49,60 @@ module "project-gcp" {
 ###############################################################################
 
 module "vpc-onprem" {
-  source     = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/net-vpc?ref=v5.0.0"
+  source     = "../../modules/net-vpc"
   project_id = module.project-onprem.project_id
-  name       = "onprem-net"
+  name       = "onprem"
   subnets = [
     {
-      ip_cidr_range      = var.onprem_ip_range
-      name               = "onprem-sub"
+      ip_cidr_range      = var.ip_ranges.onprem
+      name               = "onprem-subnet"
       region             = var.region
       secondary_ip_range = {}
     }
   ]
 }
 
 module "firewall-onprem" {
-  source               = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/net-vpc-firewall?ref=v5.0.0"
+  source               = "../../modules/net-vpc-firewall"
   project_id           = module.project-onprem.project_id
   network              = module.vpc-onprem.name
   admin_ranges_enabled = true
-  admin_ranges         = ["0.0.0.0/0"]
```

Comment:
You can use the subnet range here, or skip the attribute entirely. You don't need to specify attributes who already have empty defaults.